org.freedesktop.Sdk.Extension.ziglang.json

```diff
@@ -0,0 +1,90 @@
+{
+    "id": "org.freedesktop.Sdk.Extension.ziglang",
+    "default-branch": "19.08",
+    "branch": "19.08",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "build-extension": true,
+    "separate-locales": false,
+    "appstream-compose": false,
+    "build-options": {
+        "prefix": "/usr/lib/sdk/ziglang"
+    },
+    "modules": [
+        {
+            "name": "zig",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /usr/lib/sdk/ziglang",
+                "cp -r ./* /usr/lib/sdk/ziglang",
+                "chmod a+x /usr/lib/sdk/ziglang/zig"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ziglang.org/download/0.6.0/zig-linux-x86_64-0.6.0.tar.xz",
```

Comment:
It would be nice to use Flathub's data checker to auto update the SDK, see what we do for Rust stable.

Reply:
It seems like `"type": "rotating-url"` and `"type": "html"` are fine, but I haven't seen any URLs/webpages that can be used with those.

Reply:
Yeah, can't find them. Should I make a HTML page that just reports latest version of Zig and use it?

Reply:
if there's no way to use the auto-updater for now, it's not a problem. Just make sure to keep it up to date.